doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -612,7 +612,7 @@ Run custom drive benchmark if defaults are not accurate (manual)
 If the default OSD capacity is not accurate, the recommendation is to run a
 custom benchmark using your preferred tool (e.g. Fio) on the drive and then
 override the ``osd_mclock_max_capacity_iops_[hdd, ssd]`` option as described
-in the `Specifying  Max OSD Capacity`_ section.
+in the `OSD Capacity Determination (Automated)`_ section.
```

Comment:
The correct section title is "Set or Override Max IOPS Capacity of an OSD". Please change it accordingly.

Reply:
Thank you so much for catching this mistake!  
I missed that section since both my suggestion and your correction talk about the same configuration.

Pushed the fix, will request review again after confirmed links in the rendered PR work as intended.